src/main/scala/rocket/fpu.scala

```diff
@@ -12,8 +12,8 @@ import cde.{Parameters, Field}
 
 case class FPUConfig(
   divSqrt: Boolean = true,
-  sfmaLatency: Int = 2,
-  dfmaLatency: Int = 3
+  sfmaLatency: Int = 3,
+  dfmaLatency: Int = 4
```

Comment:
Last time you changed the defaults for the muldiv unit, I bet you would do it this way but instead you changed the entry in Configs.scala instead. Any reason for the difference this time?